Examples/FilterProgressReporting/FilterProgressReporting.py

```diff
@@ -15,57 +15,60 @@
 #  See the License for the specific language governing permissions and
 #  limitations under the License.
 #
-#=========================================================================
+# =========================================================================
 
 from __future__ import print_function
 
 import SimpleITK as sitk
 import sys
 import os
 
-if len ( sys.argv ) < 4:
-    print( "Usage: "+sys.argv[0]+ " <input> <variance> <output>" )
-    sys.exit ( 1 )
+if len(sys.argv) < 4:
+    print("Usage: " + sys.argv[0] + " <input> <variance> <output>")
+    sys.exit(1)
 
-##! [python director command]
+
+# #! [python director command]
```

Comment:
What does this comment mean? Are these directives for sphinx? The directives appear in this file in other places too.

Reply:
I think the ##! mark up may be important to sphinx.

Reply:
Oh, ok.  I can undo that.  pycodestyle says there should be a space, but we can ignore that.